src/Redux/slice/CardSlice.jsx

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const cardSlice = createSlice({
+    name : 'Cart',
+    initialState : [],
```

Comment:
declare initialState in separate variable before creating slice.
it will be like this {
isLoading: false,
isSuccess: false,
isError: false,
data:{}
}
keep any data inside data object